tests/cpptests/test_cpp_index.cpp

```diff
@@ -1083,7 +1083,11 @@ TEST_F(IndexTest, testHugeSpec) {
   s = IndexSpec_Parse("idx", (const char **)&args[0], args.size(), &err);
   ASSERT_TRUE(s == NULL);
   ASSERT_TRUE(QueryError_HasError(&err));
+#if !defined(__arm__) && !defined(__aarch64__)
   ASSERT_STREQ("Schema is limited to 128 TEXT fields", QueryError_GetError(&err));
+#else
+  ASSERT_STREQ("Schema is limited to 64 TEXT fields", QueryError_GetError(&err));
+#endif
```

Comment:
We test other flags in redisearch.h
```
#if defined(__x86_64__) && !defined(RS_NO_U128)
/* 64 bit architectures use 128 bit field masks and up to 128 fields */
typedef __uint128_t t_fieldMask;
#define RS_FIELDMASK_ALL (((__uint128_t)1 << 127) - (__uint128_t)1 + ((__uint128_t)1 << 127))
#else
/* 32 bit architectures use 64 bits and 64 fields only */
typedef uint64_t t_fieldMask;
#define RS_FIELDMASK_ALL 0xFFFFFFFFFFFFFFFF
#endif
```

Reply:
`RS_NO_U128` is a weirdo. I think we should be more explicit in redisearch.h checking for ARM values.